apps/mosaico/src/Mosaico.purs

```diff
@@ -161,12 +161,16 @@ mosaicoComponent initialValues props = React.do
   let loadFeed feedName = do
         maybeFeed <- case feedName of
           TagFeed t -> Just <<< ArticleList <$> Lettera.getByTag 0 20 t mosaicoPaper
-          CategoryFeed Nothing -> do
-            html <- map Html <<< hush <$> Lettera.getFrontpageHtml mosaicoPaper "Startsidan"
-            Just <$> maybe (ArticleList <$> Lettera.getFrontpage mosaicoPaper Nothing) pure html
-          CategoryFeed (Just c)
-            | Just cat <- unwrap <$> Map.lookup c state.catMap
-            , cat.type == Feed -> Just <<< ArticleList <$> Lettera.getFrontpage mosaicoPaper (Just $ unwrap cat.label)
+          CategoryFeed c
+            | Just cat <- unwrap <$> Map.lookup c state.catMap ->
+              let label = unwrap c
+                  getArticleList = ArticleList <$> Lettera.getFrontpage mosaicoPaper (Just label)
+              in case cat.type of
+                Prerendered ->
+                  map Html <<< hush <$> Lettera.getFrontpageHtml mosaicoPaper label >>=
```

Comment:
It's a matter of taste, but I think this would be easier for the reader if the flow would go in one direction, like
```
map Just <<<  maybe getArticleList pure =<< map Html <<< hush <$> Lettera.getFrontpageHtml mosaicoPaper label
```

Reply:
Now that you mention it, it is questionable style to mix composition directions in the same expression. I changed them to go the same way everywhere and changed the bind direction too.